bears/js/JSStandardBear.py

```diff
@@ -16,13 +18,53 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
```

Comment:
note this will also need to change for both bears, as 'JS in Markdown' is still just Markdown.